tests/bluechi_test/service.py

```diff
@@ -23,6 +23,7 @@ class Option(enum.Enum):
     Description = "Description"
     StopWhenUnneeded = "StopWhenUnneeded"
     Wants = "Wants"
+    BindTo = "BindTo"
```

Comment:
```suggestion
    BindsTo = "BindsTo"
```
https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#BindsTo=